jetty-client/src/main/java/org/eclipse/jetty/client/RoundRobinConnectionPool.java

```diff
@@ -78,10 +103,10 @@ protected Connection activate()
                 if (entry.connection == null)
                     break;
 
-                if (!entry.active)
+                if (entry.active <= getMaxMultiplex())
```

Comment:
If I'm reading the code right, it will round robing over the connections until they all have active==1, then again over all connections until they are active==2 etc.... rather than the alternative of filling up first connection before selecting next...

Reply:
That is correct.
The `<=` is wrong as it produces an off-by-one error (should just be `<`).